cmd-replay.c

```diff
@@ -234,16 +234,16 @@ static void setup_field(struct opts *opts)
 
 	if (*str == '+') {
 		/* prepend default fields */
-		add_field(field_table[0]);  /* duration */
-		add_field(field_table[1]);  /* tid */
+		add_field(field_table[REPLAY_F_DURATION]);
+		add_field(field_table[REPLAY_F_TID]);
 		s++;
```

Comment:
OK, then I think it's better to add a coment on the field_table[] that it should match to replay_field_id.